xml/dm_registers.xml

```diff
@@ -603,7 +606,7 @@
         If \Fsbasize is 0, then this register is not present.
 
         When the system bus master is busy, writes to this register will set
-        \Fsberror and don't do anything else.
+        \Fsbbusyerror and don't do anything else.
 
         \begin{steps}{If \Fsberror is 0 and \Fsbreadonaddr is set then writes
```

Comment:
I believe we need to  augment these sentences (wherever they occur) to include \Fsbusyerror being 0 as well

Reply:
Yes. Fixed.